### PR TITLE
capstone, add 5.0-rc3

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
Needs at least one more patch for the pkgconfig file.

File: dev-util/capstone/capstone5-5.0~rc3.recipe [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	cmake -Bbuild -S. $cmakeDirArgs -DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_BINDIR=$commandBinDir \
+		-DBUILD_SHARED_LIBS=ON -L
```
Comment (jmairboeck) [inline dev-util/capstone/capstone5-5.0~rc3.recipe:83]: shouldn't `-L` be removed here?

Comment (Begasus) [reply dev-util/capstone/capstone5-5.0~rc3.recipe:83]: Leftover, I always forget to remove it (doesn't harm though) :)

Comment (Begasus) [reply dev-util/capstone/capstone5-5.0~rc3.recipe:83]: Done in https://github.com/haikuports/haikuports/pull/8866

**Pull request merged**